libraries/rdiff_backup.rb

```diff
@@ -32,7 +32,7 @@ class RdiffBackup < LWRPBase
   end
   class Provider
     class RdiffBackup < LWRPBase
-      action :delete do
+      action :delete do # rubocop:disable BlockLength
```

Comment:
This breaks the jenkins linting test because its using an older version of rubocop. You can either leave this here but use two pound signs so rubocop can skip it, or just remove it for now.